openedx_external_enrollments/external_enrollments/greenfig_external_enrollment.py

```diff
@@ -0,0 +1,119 @@
+"""GreenfigInstanceExternalEnrollment class file."""
+import io
+import logging
+
+import requests
+from rest_framework import status
+
+from openedx_external_enrollments.edxapp_wrapper.get_site_configuration import configuration_helpers
+from openedx_external_enrollments.edxapp_wrapper.get_student import get_user
+from openedx_external_enrollments.external_enrollments.base_external_enrollment import BaseExternalEnrollment
+from openedx_external_enrollments.models import EnrollmentRequestLog
+
+LOG = logging.getLogger(__name__)
+
+
+class GreenfigInstanceExternalEnrollment(BaseExternalEnrollment):
+    """
+    GreenfigInstanceExternalEnrollment class.
+    """
+
+    def __init__(self):
+        """
+        """
+        self.DROPBOX_API_URL = configuration_helpers.get_value('DROPBOX_API_URL', 'https://content.dropboxapi.com/2')
+        self.DROPBOX_FILE_PATH = configuration_helpers.get_value('DROPBOX_FILE_PATH', '/courses.txt')
+        self.DROPBOX_TOKEN = configuration_helpers.get_value('DROPBOX_TOKEN', 'token')
+
+    def __str__(self):
+        """
+        """
+        return "greenfig"
+
+    def _execute_post(self, url, headers=None, json_data=None):
+        """
+        """
+        response = requests.post(
+            url=url,
+            data=json_data,
+            headers=headers
+        )
+        return response
+
+    def _get_enrollment_headers(self):
+        """
+        """
+        headers = {
+            "Authorization": "Bearer {token}".format(token=self.DROPBOX_TOKEN),
+            "Content-Type": "application/octet-stream",
+            "Dropbox-API-Arg": "{\"path\":\"%s\",\"mode\":{\".tag\":\"overwrite\"}}" % self.DROPBOX_FILE_PATH
```

Comment:
Use format to concatenation.
Trailing comma.
Will this chnage '"{\"path\":\"%s\",\"mode\":{\".tag\":\"overwrite\"}}"' in the future? @luismorenolopera